tests/test_commcare_hq_client.py

```diff
@@ -102,6 +123,14 @@ def test_iterate_date(self):
         self._test_iterate(FakeDateFormSession(), get_paginator('form'), 3, [1, 2, 3])
         self._test_iterate(FakeDateCaseSession(), get_paginator('case'), 2, [1, 2])
 
+    def test_repeat_limit(self):
+        try:
```

Comment:
nit: it might more appropriate to use something like [`pytest.raises`](https://docs.pytest.org/en/stable/reference.html?highlight=raises#pytest.raises) for this (offhand I couldn't remember what test stack we have)

Reply:
Thanks, I've just pushed that change.